### PR TITLE
made "Disable cache clearing" hidden, and removed logic for clearing …

### PR DESCRIPTION
**Pull request by MatthiasSort**
…cache as the job.cs handle it. if there is any who needs to be cleared it still rebuild the Assortments.

### CODE REVIEW

**Review by DWDBE — APPROVED** [review]

File: src/DynamicwebProvider.cs [hunk 1]
```diff
@@ -192,7 +192,7 @@ public string DefaultLanguage
     [AddInParameter("Repositories index update"), AddInParameterEditor(typeof(DropDownParameterEditor), "multiple=true;none=true;Tooltip=Index update might affect on slower perfomance"), AddInParameterGroup("Destination"), AddInParameterOrder(80)]
     public string RepositoriesIndexUpdate { get; set; }
 
-    [AddInParameter("Disable cache clearing"), AddInParameterEditor(typeof(YesNoParameterEditor), "Tooltip=This setting disables cache clearing after import\t"), AddInParameterGroup("Destination"), AddInParameterOrder(90)]
+    [AddInParameter("Disable cache clearing"), AddInParameterEditor(typeof(YesNoParameterEditor), "Tooltip=This setting disables cache clearing after import\t"), AddInParameterGroup("Hidden"), AddInParameterOrder(90)]
```
Comment (DWDBE) [inline src/DynamicwebProvider.cs:195]: maybe this should be even removed, not sure if it make sense to have it hidden

Comment (MatthiasSort) [reply src/DynamicwebProvider.cs:195]: well... the DynamicwebProvider is Public, so this would be a breaking change?

Comment (DWDBE) [reply src/DynamicwebProvider.cs:195]: then if you think it is too dangerous then make it as Obsolete() and remove the add-in attributes as there is no need to show it

Comment (MatthiasSort) [reply src/DynamicwebProvider.cs:195]: we do both Hidden and Obsolete :)

Comment (DWDBE) [reply src/DynamicwebProvider.cs:195]: maybe the Hidden can be removed? so remove the [AddInParameter] attribute at all, is it needed for any reason?

**Review by DWDBE — APPROVED** [review]

**Review by frederik5480 — APPROVED** [review]

**Pull request merged**